build/configs/rtl8730e/loadable_apps_16M/defconfig

```diff
@@ -1274,7 +1275,7 @@ CONFIG_EXAMPLES_HELLO=y
 # CONFIG_EXAMPLES_LIBTUV is not set
 # CONFIG_EXAMPLES_LWNL_SAMPLE is not set
 # CONFIG_EXAMPLES_MEMORY_FRAGMENTATION_TEST is not set
-# CONFIG_EXAMPLES_MPU_TEST is not set
+CONFIG_EXAMPLES_MPU_TEST=y
```

Comment:
There is no mpu on rtl8730e. This looks strange. How about renaming as memory protection test regardless of this PR? I mean new PR.

Reply:
Done. I added a commit for this.